npm/pkg/dataplane/ipsets/ipsetmanager.go

```diff
@@ -555,9 +555,8 @@ func validateIPSetMemberIP(ip string) bool {
 	// always guaranteed to have ip, not guaranteed to have port + protocol
 	ipDetails := strings.Split(ip, ",")
 	if util.IsIPV4(ipDetails[0]) {
-		return true
+		err := ValidateIPBlock(ipDetails[0])
```

Comment:
should we validate the whole `ip` like we used to?

Reply:
I had to change this because it would fail in the case of tcp since that method splits on a space and tcp has a comma. That method only validates CIDR so sending it already split seemed to make sense in this case